SigningService/IPackagePartSigner.cs

```diff
@@ -1,9 +1,13 @@
+using System.IO;
 using System.IO.Packaging;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
 
 namespace SigningService
 {
     internal interface IPackagePartSigner
     {
-        bool TrySign(PackagePart packagePart);
+        Task<bool> TrySignAsync(Stream peStream);
+        bool CanSign(Stream peStream);
```

Comment:
This should be async as well. I expect we'll need to check our vault to see if we have an appropriate key.